src/test-utilities/react-testing.tsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-import {createMount} from '@shopify/react-testing';
+import {createMount, mount} from '@shopify/react-testing';
 import {createThemeContext} from '../utilities/theme';
 import {ScrollLockManager} from '../utilities/scroll-lock-manager';
 import {StickyManager} from '../utilities/sticky-manager';
 import {createAppBridge} from '../utilities/app-bridge';
 import {I18n} from '../utilities/i18n';
 import translations from '../../locales/en.json';
 import {merge} from '../utilities/merge';
-import {Link} from '../utilities/link';
 import {PolarisTestProvider} from './PolarisTestProvider';
 import {WithProvidersOptions, WithProvidersContext} from './types';
 
+export {mount};
```

Comment:
Curious why we're exporting mount here? Rather than importing it direction from `react-testing`? Was it to clean up imports? (e.g. `import {mountWithContext, mount} ...`)

Reply:
Yep, exactly. I figure this file should be your one-stop-shop for stuff relating to modern testing bits. Having to export mountWithContext from here, and mount from `@shopify/react-testing` felt like there wasn't much connection between those two. This helps make it clear they're related